imports/api/courses.js

```diff
@@ -687,10 +695,34 @@ Meteor.methods({
     const previous = course.allowStudentQuestions
     Courses.update({ _id: courseId }, {
       $set: {
-        allowStudentQuestions: !previous
+        allowStudentQuestions: !previous,
+        requireApprovedQuestions: true
```

Comment:
I'm not sure this should be related to whether or not public questions require approval, right? I think you have to remove the requireApprovedQuestions here.

Reply:
This is meant to reset requireApprovedQuestions to its default value when the course instructor decides to not allow student questions. If the instructor at a later date decides to switch it back, I think the course should force the public questions to be approved by default as it is more strict. The instructor can of course allow for the questions to be unapproved again.